earthly/flutter/Earthfile

```diff
@@ -2,7 +2,7 @@ VERSION 0.8
 
 IMPORT ./installer AS installer
 
-# flutter-src - save Flutter source code as artifact.
+# Save Flutter source code as artifact.
```

Comment:
Earthly doc should begin with the target name
https://docs.earthly.dev/docs/earthly-command#earthly-doc

Reply:
I read the docs. I wouldn't say I like how Earthly does it.